README.md

```diff
@@ -1,2 +1,7 @@
+![hourly](https://github.com/enarx/bot/workflows/hourly/badge.svg)
+![cleanup-sprint-board](https://github.com/enarx/bot/workflows/cleanup-sprint-board/badge.svg)
```

Comment:
- The `hourly` workflow will be removed soon-ish. While the badge makes sense now, it likely won't in a few weeks. I'm still fine with the change, though.

- The `cleanup-sprint-board` workflow will remain, though it's only ever run manually at the end of every sprint, so it's not super helpful to have it. I'd personally drop this one.

Reply:
Will there be a new Workflow added, or is it best to just move away from these badges?